apps/src/studio/controls.html.ejs

```diff
@@ -1,16 +1,16 @@
 <% var msg = require('../locale') %>
 
 <div id="soft-buttons" class="soft-buttons-none">
-  <button id="leftButton" class="arrow">
+  <button id="leftButton" disabled=true class="arrow">
```

Comment:
Have you checked that doing this initial render with the buttons disabled does not interfere with shared apps?